slides/src/docs/asciidoc/chapters/_4_jvm.adoc

```diff
@@ -65,7 +65,9 @@ image::gradle/bg-6.png[background, size=cover]
 * Available criteria:
 ** Java language version
 ** JVM vendor
-** (Implementation: défaut or J9)
+** Implementation
```

Comment:
Me leaving French is not good, but your change is too much.
We have an `implementation` in a toolchain spec that allows `default` or `j9` but I believe that's a mistake and should not be used.